webxdc.d.ts

```diff
@@ -97,6 +97,15 @@ interface Webxdc<T> {
    * @returns returns a promise that never resolves (because the xdc closes), but is rejected on error.
    */
   sendToChat(content: sendOptions): Promise<void>;
+  /**
+   * TODO
+   */
+  importFiles(filters: {
+    mimeTypes?: string[];
+    extentions?: string[];
```

Comment:
`extensions` need a comment as being ambiguous in different contexts. eg.  as `.ext`, `*.ext` or `ext` may all be regarded as "extensions" or "extension filters".

Reply:
question is what we want to do there?
I would copy the filters of the file input field, which don't have `*.ext` and every extension needs to start with a dot: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#unique_file_type_specifiers (note that we do mime types as a separate filter because it makes implementing our own ui a bit easier. we don't have to separate them in our ui code.)

Reply:
probably support the same as the <input type=file> API

Reply:
> question is what we want to do there?

i mean, things are passed to `<input accept>`, so using that format seems natural and straight forward, there seem to be no good reasons to use sth. else. 

nb: beside advantages of using and filter-object with two arrays (easier webxdc implementations - what else?), when reading the code snippet, i see the disadvantage that using `importFiles` as an alternative to `<input accept>` is a little bit harder. just saying, i would be fine with both.

Reply:
> i see the disadvantage that using importFiles as an alternative to <input accept> is a little bit harder for the xdc user

I don't really see how it is harder? It saves the user from writing the code that we have in the emulator.
It also makes clear what you can write there without reading the file input api.

If you are too lazy to write an array like `[".jpg",".png",".gif"]` then you can always also write `".jpg,.png,.gif".split(","`

Reply:
> I don't really see how it is harder?

assume, you are using some library that uses `<input accept="application/msword,.doc,.docx,application/vnd.ms-excel,.xls,.xlsx,application/vnd.ms-powerpoint,.ppt,.pptx">` or even more complicated stuff and you want to port that library using webxdc.

even the simplest conversions _is_ harder and more error prone then just being allowed to use _exactly_ the same string (and the conversion is not just a .split() as you have to sort that into different arrays as well).

i am not saying that we have to go that way, i am +.0 here, but i am not buying that it is not harder.

in general, it is always a good idea to pick up existing signatures and parameter patterns, esp. when doing things that are very close to existing things - therefore my questions for the two arrays: if it is used already in another api that many ppl already know, then it is another story.

Reply:
it would just open the native file picker and not our file picker then. If you change the library then you can attach before the mimetypes and extensions get combined.

also it's not that hard: everything starting with a dot is an extension, anything else containing a slash is a mime type.